service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/AddCaseNumberController.java

```diff
@@ -45,7 +47,11 @@ public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
         CaseData previousData = mapper.convertValue(callbackRequest.getCaseDetailsBefore().getData(), CaseData.class);
 
         if (isEmpty(previousData.getFamilyManCaseNumber())) {
-            applicationEventPublisher.publishEvent(new CaseNumberAdded(callbackRequest.getCaseDetails()));
+            try {
+                applicationEventPublisher.publishEvent(new CaseNumberAdded(callbackRequest.getCaseDetails()));
+            } catch (Exception exc) {
+                log.error("Robotics notification failed", exc);
+            }
```

Comment:
i think this one is not really needed as it will be logged anyway. I rather though that there should be log when controller does not sent notification due to if condition

Reply:
updated